### ISSUE TITLE
Error messages are inconsistent

### ISSUE THREAD

**Issue by solidpixel** [ARM-software/astc-encoder#100]
The error messages omitted by the CLI are inconsistent; 

* Some are just sentences.
* Some have "Error:" prefix.
* Some have "ERROR: "prefix. 

Low priority cleanup task to standardize them. Once this is done we can enable a functional test check for "ERROR" in the output for negative tests; currently we only check return codes because of the difficulty in checking inconsistent logs.

**Comment by solidpixel** [comment]
Fixed and merged into master.
